integration-tests/src/main/java/org/apache/druid/testing/utils/QueryResultVerifier.java

```diff
@@ -25,12 +25,13 @@
 import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
+import java.util.Optional;
 
 public class QueryResultVerifier
 {
   private static final Logger LOG = new Logger(QueryResultVerifier.class);
 
-  public static boolean compareResults(
+  public static Optional<String> compareResults(
```

Comment:
Nit: As I see this method being used a lot, lets javadoc this

Reply:
+1